app/views/invoices/show.html.erb

```diff
@@ -17,3 +17,7 @@
     </li>
   <% end %>
 </ul>
+
+<div id="total_invoice_revenue">
+<p>Total Revenue From This Invoice: $<%= (@invoice.calculate_invoice_revenue / 100.to_f).round(2)%></p>
```

Comment:
Should we maybe write a method that converts it to the decimal form in the model? just to keep it contained and dry it up?